modules/color-picker/main/index.coffee

```diff
@@ -227,7 +227,7 @@ class ColorPicker extends hx.EventEmitter
           if endCallback isnt null then endCallback(elem)
           dragging = false
 
-        dispose: ->
+        dispose = ->
```

Comment:
It looks like `dispose()` and `disposed` are not used any more so this function can possibly be removed entirely